src/page-response-impact/ResponseImpactDashboard.tsx

```diff
@@ -163,37 +219,97 @@ const ResponseImpactDashboard: React.FC = () => {
                   Back to model
                 </BackDiv>
               </Link>
-              <ScenarioName>{scenario?.name}</ScenarioName>
-              <PageHeader>COVID-19 Response Impact as of [DATE]</PageHeader>
-              <SectionHeader>Safety of Overall Population</SectionHeader>
+              <PageHeader>
+                [State or County] COVID-19 Response Impact
```

Comment:
Question: I'm not sure how to get the `stateCode` or `countyName` on this page. 

Is there a way to console.log or browse what I have available to me model-wise? Not sure what I'm missing yet

Once have it, it would be something like this?

```
{stateCode ? ${stateCode} State Prison : ${countyName} County Jail} COVID-19 Response
```

Reply:
I could be mistaken, but I believe you will find them in the useLocaleDataState at the top of this component. 

As for the copy, I believe this page is modelling across an entire scenario, right @sychang ? If so, state and county are concepts at the facility level, not the scenario level. So unless we have validated that all facilities share a locale, putting them in the title is not totally sensible. We should also keep in mind that county is optional and you'd likely have different counties on different facilities, even if they all share the same state.

Reply:
The`localeDataSource` is an object which holds the static locale data which comes from the [Excel model](https://docs.google.com/spreadsheets/d/1tsLJch_r9mSMAK3LxpMER8lgsAeytUSNXO6Zpksg1gQ/edit#gid=1836987932). You can use `getLocaleDefaults` to see the locale data for a specific state/county.

The state code and county name are available on the facilities, and therefore the `modelInputs`. We are doing a check in the `ValidDataWrapper` that the state codes are consistent across all facilities, and if the system type is 'County Jail', that the county name is consistent across all facilities. So while this model is at the scenario level, it is safe to use the state code from the first facility. I believe that @sychang has updated the requirement for the page title for MVP so you will want to check with her before putting too much effort into this title.

You can add this console.log on line 207 and see the `stateCode` and `countyName`

`modelInputs[0] && console.log('stateCode:', modelInputs[0].stateCode, 'countyName:', modelInputs[0].countyName)`

Or this option to see the locale data for the give state code and county name.

`console.log(modelInputs[0] && getLocaleDefaults(localeDataSource, modelInputs[0].stateCode, modelInputs[0].stateCode))`

Reply:
Thanks, got access to the Excel.

I think this is what you mean? https://github.com/Recidiviz/covid19-dashboard/pull/335/commits/81900bc5d439a006fee530749ea1f26db6e197f7

`County Jail`:
![image](https://user-images.githubusercontent.com/1372946/81341451-6b54b980-9066-11ea-83ed-9fe5ecd7ab14.png)

`State Prison`:
![image](https://user-images.githubusercontent.com/1372946/81341516-87585b00-9066-11ea-85c4-4c647d4d0597.png)

//

Not sure how to get the `State Jail` option locally? But can figure that out separately

![image](https://user-images.githubusercontent.com/1372946/81341703-d56d5e80-9066-11ea-92fc-d78992276cad.png)